lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -61,6 +68,10 @@ def self.auto_generate_changelog(repo_name, github_token, rate_limit_sleep)
             section = get_section_depending_on_types_of_change(types_of_change)
 
             line = "* #{message} via #{name} (@#{username})"
+            if message.include?(Fastlane::Actions::BumpPhcVersionAction::PR_TITLE)
+              # Append links to native releases
+              line += native_releases_links(github_token, hybrid_common_version, versions_file_path)
```

Comment:
So if we bump PHC multiple times for the same version (shouldn't happen often) this will add the same entries multiple times right? 

Thinking of possible solutions, I guess we should do this not in a per-commit basis but just getting the last phc bump since the last release and do the comparison. It might be tricky though. What do you think?

Reply:
I've actually encountered that in the past and I manually removed the extra changelog line to prevent something like:

```
* [AUTOMATIC BUMP] Updates purchases-hybrid-common to 4.5.3 (#553) via RevenueCat Git Bot (@RCGitBot)
* [AUTOMATIC BUMP] Updates purchases-hybrid-common to 4.5.2 (#553) via RevenueCat Git Bot (@RCGitBot)
```

This might be a good opportunity to fix it. I wasn't sure how to do it but if we start using `phc_dependencies`, we might be able to do it easily

Reply:
I think we can do this as a followup if you don't mind